prooftools/representationproof.go

```diff
@@ -0,0 +1,41 @@
+package prooftools
+
+import (
+	"github.com/privacybydesign/gabi/big"
+	"github.com/privacybydesign/gabi/keyproof"
+)
+
+type QrRepresentationProofStructure keyproof.RepresentationProofStructure
+
+func (s *QrRepresentationProofStructure) CommitmentsFromSecrets(g *PublicKeyGroup, list []*big.Int, bases keyproof.BaseLookup, secretdata keyproof.SecretLookup) []*big.Int {
```

Comment:
Here and in the methods below, I think it would be easier on the caller to accept a `*keys.PublicKey` instead of a `*PublicKeyGroup`, and cast it ourselves to the latter here.

Reply:
fixed